test/test.config.js

```diff
@@ -269,4 +269,16 @@ describe('Config API', function() {
         expect(configResult.toString()).to.be(error);
         done();
     });
+
+    it('should throw error on config.storeName not string', function(done) {
```

Comment:
Just a wording suggestion here to clean up the test:

```suggestion
    it('should throw error if config.storeName is not a string', function() {
```

(You also don't need to use `done()` here since this isn't an asynchronous test.)

Reply:
Ok